src/app/containers/FeatureIndexMain/index.jsx

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable jsx-a11y/aria-role */
+import React from 'react';
+import path from 'ramda/src/path';
+import { frontPageDataPropTypes } from '#models/propTypes/frontPage';
+import MetadataContainer from '../Metadata';
+import LinkedData from '../LinkedData';
+import ATIAnalytics from '../ATIAnalytics';
+import ChartbeatAnalytics from '../ChartbeatAnalytics';
+import StoryPromoGroups from '../StoryPromoGroups';
+
+const FeatureIndexMain = ({ featureIndexData }) => {
+  const title = path(['metadata', 'title'], featureIndexData);
+  const groups = path(['content', 'groups'], featureIndexData);
+  const lang = path(['metadata', 'language'], featureIndexData);
+  const description = path(['metadata', 'summary'], featureIndexData);
+  const seoTitle = path(['promo', 'name'], featureIndexData);
+
+  return (
```

Comment:
Pretty much everything outside of the `<main>` tags could be shared with the Front Page as is (something like this):

```
  const IndexPage = ({ title, data, children }) => {
  const lang = path(['metadata', 'language'], data);
  const description = path(['metadata', 'summary'], data);
  const seoTitle = path(['promo', 'name'], data);  return (
    <>
      <ATIAnalytics data={data} />
      <ChartbeatAnalytics data={data} />
      <MetadataContainer
        title={title}
        lang={lang}
        description={description}
        openGraphType="website"
      />
      <LinkedData type="WebPage" seoTitle={seoTitle} />
      <main role="main">{children}</main>
    </>
  );
};
```

Might be better to not go over the top with making things generic though, as I assume there's more differences to come between feature indexes and the front page

Reply:
There may well be more differences to come with feature index vs front pages but we do not know those yet unfortunately

Reply:
Yeah, it's not a large amount of repeated code so I think it's safer to keep them separate rather than have to unpick a shared component as soon as there's a divergence